src/MartinGeorgiev/Doctrine/DBAL/Types/Point.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MartinGeorgiev\Doctrine\DBAL\Types;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use MartinGeorgiev\Doctrine\DBAL\Types\Exceptions\InvalidPointForDatabaseException;
+use MartinGeorgiev\Doctrine\DBAL\Types\Exceptions\InvalidPointForPHPException;
+use MartinGeorgiev\ValueObject\Point as PointValueObject;
+
+/**
+ * Implementation of PostgreSQL POINT data type.
+ *
+ * @see https://www.postgresql.org/docs/current/datatype-geometric.html#DATATYPE-GEOMETRIC-POINTS
```

Comment:
```suggestion
 * @see https://www.postgresql.org/docs/17/datatype-geometric.html#DATATYPE-GEOMETRIC-POINTS
```

It's better to use a specific version as the `current` pages are dynamic, and this link may not be live in the future.